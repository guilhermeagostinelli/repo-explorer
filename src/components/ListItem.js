import styled from "styled-components";

const ListItem = styled.li`
  padding: 12px;
  text-align: center;
`;

export default ListItem;
