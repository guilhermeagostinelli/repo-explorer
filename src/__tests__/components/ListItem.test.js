import React from "react";
import { create } from "react-test-renderer";
import ListItem from "../../components/ListItem";
import "jest-styled-components";

describe("<ListItem />", () => {
  it("should match the snapshot", () => {
    const listItem = create(<ListItem />);
    expect(listItem.toJSON()).toMatchSnapshot();
  });
});
