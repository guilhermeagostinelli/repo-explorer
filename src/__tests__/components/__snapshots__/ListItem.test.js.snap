// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ListItem /> should match the snapshot 1`] = `
.c0 {
  padding: 12px;
  text-align: center;
}

<li
  className="c0"
/>
`;
